src/core/default/mappers/simple/relational_db_deployer.py

```diff
@@ -39,7 +39,7 @@ def _create_simple_relational_db(
             "ScalingConfiguration": {
                 "MinCapacity": resource.MinCapacity,
                 "MaxCapacity": resource.MinCapacity,
-                "AutoPause": resource.SecondsToPause == 0,
+                "AutoPause": not resource.SecondsToPause == 0,
```

Comment:
It's recommended to use != instead of not ==
<img width="840" alt="image" src="https://user-images.githubusercontent.com/2273155/173199062-31b430ab-8311-4a04-9847-801e8ff8a16a.png">

Reply:
Sounds good Ill change it and merge